.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+# Keep GitHub Actions up to date with GitHub's Dependabot...
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    groups:
+      GitHub_Actions:
```

```suggestion
      actions:
```

Pretty sure this is `actions`, I've been using that in lots of repos, like https://github.com/scientific-python/repo-review/blob/79911cd34aaec4092d467a115ef69f238ee08230/.github/dependabot.yml#L9

---

Line 6 ___needs___ to say `github-actions` but line 9 is just a label so I sense that it can follow local conventions.  I made the requested change.

---

Ahh, okay.